### ISSUE TITLE
[cetic/nifi] kubectl port-forward: connection refused

### ISSUE THREAD

**Issue by jenisov** [cetic/helm-nifi#46]
**Describe the bug**
Cannot get access to ui when deployed to microk8s.

**Version of Helm and Kubernetes**:
helm 2.16.0
kubernetes 1.17.0 (microk8s latest)

**What happened**:
kubectl port-forward is getting connection refused from nifi service port 80

**What you expected to happen**:

**How to reproduce it** (as minimally and precisely as possible):
`$ kubectl port-forward service/nifi 8080:80 -n nifi
Forwarding from 127.0.0.1:8080 -> 8080
Forwarding from [::1]:8080 -> 8080
Handling connection for 8080
E0128 20:54:25.693343   19817 portforward.go:400] an error occurred forwarding 8080 -> 8080: error forwarding port 8080 to pod e4b0944121d6dd616ce42a46210581b344b3196a09242f784b098fa141c7cc0f, uid : failed to execute portforward in network namespace "/var/run/netns/cni-caf78bef-7782-c265-94fc-05dd2e49672a": socat command returns error: exit status 1, stderr: "2020/01/28 20:54:25 socat[20834] E connect(5, AF=2 127.0.0.1:8080, 16): Connection refused\n"`

**Anything else we need to know**:
setting webProxyHost in values.xml doesn't help

**Comment by CCsamteale** [comment]
Hitting the same issue as above at the moment, looks to be related to kubernetes issue
https://github.com/kubernetes/kubernetes/issues/72597

Nifi listens on the PodIP, but port-forward talks to 127.0.0.1

**Comment by jenisov** [comment]
I found workaround but it requires Ingress installed to the cluster:

```
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
   name: nifi-ingress
spec:
   rules:
   - host: localhost
     http:
       paths:
       - path: /nifi
         backend:
           serviceName: my-nifi
           servicePort: 80
   - host: my.host.com
     http:
       paths:
       - path: /nifi
         backend:
           serviceName: my-nifi
           servicePort: 80
```

**Comment by alexnuttinck** [comment]
It should be fixed now (in Chart v0.4.2), please repoen if you still encounter this issue.
